packages/core/contracts/financial-templates/perpetual-multiparty/ConfigStore.sol

```diff
@@ -153,24 +161,34 @@ contract ConfigStore is ConfigStoreInterface, Testable, Lockable, Ownable {
 
     // Use this method to constrain values with which you can set ConfigSettings.
     function _validateConfig(ConfigStoreInterface.ConfigSettings memory config) internal pure {
-        // Its hard to ascertain what are reasonable limits, into the future and the past, for proposal timestamps.
-        // So, we do not set any limits.
+        // We don't set limits on proposal timestamps because there are already natural limits:
```

Comment:
Should we just remove the future limit from the config now that the OO enforces a future limit, and this limit would never be hit?

Reply:
Done!